configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionRepository.java

```diff
@@ -665,7 +687,7 @@ public Lock lock(long sessionId) {
         return curator.lock(lockPath(sessionId), Duration.ofMinutes(1)); // These locks shouldn't be held for very long.
     }
 
-    private Path lockPath(long sessionId) {
+    public Path lockPath(long sessionId) {
```

Comment:
intended? I don't see any new usage in this PR.

Reply:
I tried to write a test that needed this (but that was abandoned), so not anymore, will fix